docs/software/python.md

```diff
@@ -215,13 +209,13 @@ The command above activates the base environment provided by Mamba, which includ
 (base) [johndoe@c3cpu-a7-u19-1 ~]$
```

Comment:
we usually avoid mentioning specific versions of things, so we don't have to update the documentation when we update the module.

If you feel this is important to include, include the version number in the module load command OR say something to the effect of "at the time of writing, the default Mamba version provides Python 3.9.16" or perhaps give exclude mentioning the python version at all.